examples/simple.c

```diff
@@ -131,6 +132,8 @@ int main(void)
     glVertexAttribPointer(vcol_location, 3, GL_FLOAT, GL_FALSE,
                           sizeof(vertices[0]), (void*) (sizeof(float) * 2));
 
+    glfwGetMonitorWorkarea(glfwGetPrimaryMonitor(), &workarea_x, &workarea_y, &workarea_width, &workarea_height);
+    printf("Monitor work area: %d, %d, %d, %d\n", workarea_x, workarea_y, workarea_width, workarea_height);
```

Comment:
This would probably be better in another example than the simple one, which is meant to be simple.

Reply:
Moved the example/test to monitors.c: https://github.com/dougbinks/glfw/commit/a484f0d8ddba3c4a1f7aaf12858812a5c0f64607